xcode/Subconscious/Shared/Components/AppTabView.swift

```diff
@@ -19,7 +19,7 @@ struct AppTabView: View {
                 .tabItem {
                     Label("Feed", systemImage: "newspaper")
                 }
-            NotebookView(parent: store)
+            NotebookView(app: store)
```

Comment:
Converging on a consisten naming scheme where the app store is always called `app`.